src/main/webapp/util/uccConfigurationValidators.js

```diff
@@ -64,6 +64,17 @@ export const parseFunctionRawStr = (rawStr) => {
     return { error, result };
 };
 
+export const parseFileValidator = (data, validFileTypes) => {
+
+    const fileName = data.split(",")[0].split(".");
+    const extention = "." + fileName[fileName.length - 1];
```

Comment:
extention -> extension

this line and line below, typo

Reply:
done